dask_geomodeling/raster/parallelize.py

```diff
@@ -0,0 +1,123 @@
+"""
+Module containing blocks that parallelize raster blocks
+"""
+from itertools import product
+import numpy as np
+
+from .base import BaseSingle
+
+
+__all__ = ["RasterTiler"]
+
+
+class RasterTiler(BaseSingle):
+    """Parallelize operations on a RasterBlock by tiling the request.
+
+    The tiles are computed starting at the topleft corner of the requested box.
```

Comment:
Box is probably "bbox" here, and the code starts at the lower y, therefore this seems not correct. Maybe better to leave it out completely since it is also not very relevant to the block user.